Demos/CMakeLists.txt

```diff
@@ -124,29 +127,31 @@ macro(make_silkit_demo executableName demoSourceFile)
             SilKit::SilKit 
             Threads::Threads
     )
-
+    target_include_directories(${executableName} PRIVATE ${SILKIT_DEMO_DIR}/include)
```

Comment:
```suggestion
    target_include_directories(${executableName} PRIVATE ${SILKIT_DEMO_DIR}/include)

```

Reply:
I moved the include folder into `Demos/communication` and created another macro `make_silkit_communication_demo` 
 to not sprinkle the include dir unnecessarily into the other demos.